client/profile/user-profile-overlay.tsx

```diff
@@ -1,14 +1,20 @@
 import React, { useCallback } from 'react'
 import { User } from '../../common/users/user-info'
 import MenuItem from '../material/menu/item'
-import { Popover, useAnchorPosition } from '../material/popover'
+import { OriginX, OriginY, Popover, useAnchorPosition } from '../material/popover'
 import { useAppSelector } from '../redux-hooks'
 import { navigateToWhisper } from '../whispers/action-creators'
 import { Actions, Header, PopoverContents, StyledAvatar, Username } from './profile-overlay-content'
 
 export interface PopoverRelatedProps {
   open: boolean
   anchor: HTMLElement | null
+  anchorOriginX: OriginX
```

Comment:
is there a reason we can't just use `PopoverProps` here? Maybe `PopoverProps` with an `Omit` if you really need to remove something? (but it seems like they're the same, just some stuff is missing because it didn't get added)

Also get rid of the `anchorOffset` stuff please, that stuff was a bad idea imo and it's a lot better to do the math at the point that is setting the position